### PR TITLE
Corrected ReaperScansFr

### PR DESCRIPTION
**Pull request by onelots**
**ReaperScansFr** moved to a new template / new URL so here is the patch.

### CODE REVIEW

File: src/web/mjs/connectors/ReaperScansFR.mjs [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
-        this.url = 'https://reaperscans.fr';
-        this.path = '/manga/?list';
+        this.url = 'https://new.reaperscans.fr';
+        this.links = {
+            login: 'https://new.reaperscans.fr/login'
+        };
+        this.queryChapters = 'div.chapter-link > a';
+        this.queryChaptersTitleBloat = 'span.chapter-release-date';
     }
-}
```
Comment (Sheepux) [inline src/web/mjs/connectors/ReaperScansFR.mjs:13]: Missing closing brace

**Review by Sheepux — CHANGES_REQUESTED** [review]

**Comment by onelots** [comment]
Yep, changed

File: src/web/mjs/connectors/ReaperScansFR.mjs [hunk 2]
```diff
@@ -1,13 +1,17 @@
-import WordPressMangastream from './templates/WordPressMangastream.mjs';
+import WordPressMadara from './templates/WordPressMadara.mjs';
 
-export default class ReaperScansFR extends WordPressMangastream {
+export default class ReaperScans extends WordPressMadara {
```
Comment (Sheepux) [inline src/web/mjs/connectors/ReaperScansFR.mjs]: do not change the class name, should be the same as the id and correlated to the name/label.

**Comment by onelots** [comment]
Yeah, excuse me X) i've made a mistake indeed; but now, it's fine.

**Review by Sheepux — APPROVED** [review]
/LGTM
(no problem, that's why review are here for)

**Pull request merged**